eager-workflow-start/README.md

```diff
@@ -0,0 +1,10 @@
+### Steps to run this sample:
```

Comment:
Can we add a short description here about eager workflow start and how the sample works like some of our other samples?

Reply:
Done